runtime-sdk/src/context.rs

```diff
@@ -271,6 +275,31 @@ impl<'a, 'b> TxContext<'a, 'b> {
             .map(|x| x.downcast().expect("type should stay the same"))
             .unwrap_or_default()
     }
+
+    /// Fetches or sets a value associated with the transaction.
+    pub fn tx_value<V>(&mut self, key: &'static str) -> &mut V
```

Comment:
```suggestion
    pub fn tx_value<V>(&mut self, key: &str) -> &mut V
```
I think that the static requirement is a bit heavy, but w/e

Reply:
could this apply to existing `values` related APIs too? we could open a separate issue if the scope is thus larger

Reply:
Yes this would apply to the existing APIs as well. No strong preference, but in practice these keys will have static lifetime as they will be defined as constants.

Reply:
thx. moving to #104